common/src/main/java/com/microsoft/identity/common/internal/platform/IDevicePopManager.java

```diff
@@ -146,6 +146,35 @@ public String toString() {
         }
     }
 
+    /**
+     * Information about the backing of the underlying keystore.
+     */
+    enum SecureHardwareState {
```

Comment:
Also, since this is interface-level, do we want to reserve a keyword for TRUE_ATTESTED or something?

Reply:
It just makes things easier if we retain interface compatibility when launching a new feature.  So it's worth taking a second here and making sure that we're providing the whole space of possibilities for this value even if they're not available yet.  They should pick up that documentation though.  Unfortunate that we can't use @Experimental, thought that's not strong enough.

Reply:
23bec7130d997b92d507b8c01656055974060bfc